layouts/BeltDocsLayout.re

```diff
@@ -94,23 +94,48 @@ module Md = {
     );
 };
 
+module FlavourSwitch = {
+  open BeltDocsFlavour;
+  [@react.component]
+  let make = () => {
+    let flavourContext = useContext();
+
+    <div className="flavour-switch">
```

Comment:
We use [TailwindCSS](https://tailwindcss.com) to format styles. Since it's a standalone component, we should adhere to Tailwind. For the `active` part, you can do a conditional and append to className like this:

```
let active = flavour == Flavour.Reason ? " bg-white text-black" : "";

<div className="text-white " ++ active/> 
```